### PR TITLE
🐮🥛 Butterstick Support 🥛🐮

### PR DESCRIPTION
**Pull request by germ**
Adds support for the Butter Stick from g Heavy Industries

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

Did you know it takes 21.2 pounds of whole milk to make one pound of butter?

### CODE REVIEW

File: keyboards/butterstick/butterstick.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/* Copyright 2019 %YOUR_NAME%
```
Comment (mechmerlin) [inline keyboards/butterstick/butterstick.c]: Please put your name here

File: keyboards/butterstick/butterstick.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "quantum.h"
+
+#define BUTTER_LAYOUT( 										\
```
Comment (mechmerlin) [inline keyboards/butterstick/butterstick.h]: lower case descriptions and the other way around is the format here

```suggestion
#define LAYOUT_butter( 										\
```

File: keyboards/butterstick/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,184 @@
+#include QMK_KEYBOARD_H
+
+#include "butterstick.h"
```
Comment (mechmerlin) [inline keyboards/butterstick/keymaps/default/keymap.c]: you don't need this one
```suggestion

```

Comment (germ) [reply keyboards/butterstick/keymaps/default/keymap.c]: Fixed in https://github.com/qmk/qmk_firmware/pull/5742/commits/ed263ea00aa379ffd144ad9219f682cddeaadda0

File: keyboards/butterstick/butterstick.h (hunk 2 above)
Comment (germ) [reply keyboards/butterstick/butterstick.h]: Fixed in https://github.com/qmk/qmk_firmware/pull/5742/commits/ed263ea00aa379ffd144ad9219f682cddeaadda0

File: keyboards/butterstick/butterstick.c (hunk 1 above)
Comment (germ) [reply keyboards/butterstick/butterstick.c]: Fixed in https://github.com/qmk/qmk_firmware/pull/5742/commits/ed263ea00aa379ffd144ad9219f682cddeaadda0

**Pull request merged**